.circleci/config.yml

```diff
@@ -1,4 +1,28 @@
-version: 2
+version: 2.1
+
+commands:
+  export-js-bridge:
+    parameters:
+      platform:
+        description: Platform to export - android or ios
+        type: string
+    steps: 
+      - run: 
+          name: Publish <<parameters.platform>>
+          command: |
+            set -e
+            git checkout js-bridge-<<parameters.platform>>
+            cp js-miniapp-bridge/build/src/<<parameters.platform>>/bridge.js ./
```

Comment:
This one `js-miniapp-bridge/build/` is put in `persist_to_workspace` so is it still existed even we do `git checkout branch`?

Reply:
Yeah, it should be because the `build` folder is part of `.gitignore`, so it should still be there when switching branches. I tested it on my fork, and it seemed to work: 
https://app.circleci.com/pipelines/github/corycaywood/js-miniapp/44/workflows/5e203dec-179c-4840-8412-4fc0ff981ad9/jobs/92